pom.xml

```diff
@@ -88,24 +88,14 @@ THE SOFTWARE.
 					</compilerArgs>
 				</configuration>
 			</plugin>
-			<plugin>
-				<artifactId>maven-enforcer-plugin</artifactId>
-				<configuration>
-					<rules>
-						<requireJavaVersion>
-							<version>[1.7,1.8)</version>
```

Comment:
Not sure what it was about.
1.625.3 (and actually 1.609) can be built with Java 8 IIRC